nengo/networks/assoc_mem.py

```diff
@@ -137,6 +133,100 @@ def __init__(self, input_vectors, output_vectors=None,  # noqa: C901
         self.add_input_mapping("input", input_vectors, input_scales)
         self.add_output_mapping("output", output_vectors)
 
+    @staticmethod
+    def linear_func(x_shift=0.0, x_scale=1.0):
+        """Returns a linear mapping function.
+
+        Returns a linear mapping function for ease of use when creating output
+        mappings. The returned function takes in a non-optional 'x' parameter,
+        as well as optional parameters listed below.
+
+        Parameters
+        ----------
+        x_shift: float
+            Amount to shift the linear map on the 'x'-axis.
+            E.g. If the value of x_shift = 0.3, then the output of the
+                 mapping function would be 0 at x == 0.3.
+        x_scale: float
+            Scaling factor to be applied to the 'x' input. Affects the slope
+            of the linear mapping function.
+        """
+        return lambda x, x_shift=x_shift, x_scale=x_scale: \
+            (x_scale * x) - x_scale * x_shift
```

Comment:
Oh yeah. I changed it for the other two functions. Forgot to change it for this one. :laughing: